docs/section19/19-5.md

```diff
@@ -0,0 +1,33 @@
+# Section 19 - Rule 19-5      
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True      
+**Rule ID:** 19-5       
+**Rule Description:** Unmet load hours for the proposed design shall not exceed 300 (of the 8760 hours simulated).   
+**Rule Assertion:** Options are Pass/Fail     
+**Appendix G Section:** G3.1.2.3        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each Proposed Design RulesetModelInstance  
+
+**Applicability Checks:**  
+
+1. Applies only to the proposed design rule set model instance.  
+
+**Function Calls:**  None
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`  
+
+        **Rule Assertion:** 
+        - Case 1: If the heating + cooling unmet load hours is 300 or less then pass: `if unmet_load_hours_heating + unmet_load_hours_cooling <= 300: outcome = "PASS"`   
+        - Case 2: Else, fail: `Else: outcome = "Fail"`  
```

Comment:
If during a given hour there are both unmet heating and unmet cooling loads, the hour will be conted under both  unmet_load_hours_heating AND unmet_load_hours_cooling, so will be double-counted. To address this, I suggest modifying Case to FAIL if >600 and adding Cased 3 for 301-600 unmet hours with UNDETERMINED outcome and a clarifying message. Another option is to add coincident_unmet_load_hours to the schema and base the RDS on that value.